### PR TITLE
Formatting handlebars partials

### PR DESCRIPTION
**Pull request by Mira-Alf**
## Description
* One Common twitter template for both epub and pdf generation, similar structure in place for github and hackernews
* Using partials to evaluate the pdf/epub content - the page structures are different for pdf and epub - so needed different renderTemplate processes
* Needs some testing

# Things to do
* Testing framework
* Some twitter weird scenarios cropped up
* Maybe some formatting changes needed in the templates

## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by PuruVJ** [comment]
Seems good.

File: twindle-cli/src/renderer/templates/twitter/Toc.hbs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="<%- lang %>"
+  lang="<%- lang %>">
+
+<head>
+  <title><%= title %></title>
+  <meta charset="UTF-8" />
+<<<<<<< Updated upstream:twindle-cli/src/renderer/epub/twitter/Toc.hbs
```
Comment (PuruVJ) [inline twindle-cli/src/renderer/templates/twitter/Toc.hbs:9]: Wait what is this?

**Review by PuruVJ — CHANGES_REQUESTED** [review]

File: twindle-cli/src/renderer/templates/twitter/Toc.hbs (hunk 1 above)
Comment (PuruVJ) [reply twindle-cli/src/renderer/templates/twitter/Toc.hbs:9]: This is from a merge conflict. Please resolve this part

**Comment by Mira-Alf** [comment]
Sorry @PuruVJ I missed that part

**Pull request merged**